### DISCUSSION TITLE
Project reports in OpenDCIM 23.04 blank web page, no excel

### DISCUSSION THREAD

**Discussion by alex001x** [opendcim/openDCIM#1494] [Q&A] [answered]
Hello,

The project reports in OpenDCIM 23.04 that include racks or equipment associated with the project are not functioning properly, resulting in a blank web page and no Excel file available for download. However, when I select a project without any associated racks or equipment, it does provide a downloadable Excel file with a header, a report generated by "user", and notes, but the rest is empty.

Does anyone have an idea to make it work ?
Alex.

**Comment by LFrank2021** [comment]
Hi,
most of the time you should find information in the error_log of your WebServer for example /var/log/apache2/error_log
Maybe you will see some php:warn or php:error messages pop up, as soon as you execute the report.
Kind regards
Frank

**Comment by wilpig** [comment, marked as answer]
https://github.com/opendcim/openDCIM/issues/1495

**Comment by alex001x** [comment]
OK THKS !
